### DISCUSSION TITLE
anyway to use stdlib dataclasses with config?

### DISCUSSION THREAD

**Discussion by trim21** [pydantic/pydantic#10861] [Question] [answered]
in the docs it's only possible to use config with `pydantic.dataclasses` but not stdlib dataclasses, and also `TypeAdapter` disallow add config if it's stdlib dataclasses.

It it possible to add config to stdlib dataclasses?

**Comment by Viicos** [comment, marked as answer]
See https://docs.pydantic.dev/dev/concepts/config/#configuration-with-dataclass-from-the-standard-library-or-typeddict.

**Reply by trim21** [reply]
thanks
